mqt/travis_helpers.py

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+"""
+helpers shared by the various QA tools
+"""
+
+
+RED = "\033[1;31m"
+GREEN = "\033[1;32m"
+YELLOW = "\033[1;33m"
+YELLOW_LIGHT = "\033[33m"
+CLEAR = "\033[0;m"
+
+
+def colorized(text, color):
+    return '\n'.join(
+        map(lambda line: color + line + CLEAR, text.split('\n')))
```

Comment:
```return '\n'.join([color + line + CLEAR for line in text.splitlines()])```

Reply:
Remember it is a copy-paste of original code. considering.